libs/template/chunk/src/lib/tests/chunk.directive.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Subject } from 'rxjs';
+import { ChunkModule } from '../chunk.module';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'chunk-test',
+  template: `
+    <div
+      *rxChunk="
+        strategy;
+        renderCallback: renderCallback;
+        suspenseTpl: withSuspense ? suspense : null
+      "
+    >
+      chunked
+    </div>
+    <div>not-chunked</div>
+    <ng-template #suspense>suspended</ng-template>
+  `,
+})
+class ChunkTestComponent {
+  strategy? = undefined;
+  renderCallback = new Subject<void>();
+  withSuspense = false;
+}
+
+describe('ChunkDirective', () => {
+  let fixture: ComponentFixture<ChunkTestComponent>;
+  let componentInstance: ChunkTestComponent;
+  let nativeElement: HTMLElement;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [ChunkModule],
+      declarations: [ChunkTestComponent],
+    });
+    fixture = TestBed.createComponent(ChunkTestComponent);
+    componentInstance = fixture.componentInstance;
+    nativeElement = fixture.nativeElement;
+  });
+
+  describe.each([
+    [undefined] /* <- Invalid strategy should fallback. */,
+    [''] /* <- Same here. */,
+    ['invalid'] /* <- Same here. */,
+    ['immediate'],
+    ['userBlocking'],
+    ['normal'],
+    ['low'],
```

Comment:
If we plan to work with not concurrent strats, should we test it?

Reply:
Yes we should test normal too

Reply:
done